.github/workflows/pytest.yml

```diff
@@ -29,3 +29,4 @@ jobs:
                   USERS_TABLE_NAME: users
                   VISITS_TABLE_NAME: visits
                   DOMAIN_NAME: https://visit.cumaker.space
+                  ENV: Beta
```

Comment:
out of curiosity, why ENV: Beta with the prod domain name?

Reply:
At the time I was just trying to set a variable that would pass the pytest command, and I didn't think there would be a functional difference between "Beta" and "Prod." However, looking at it now, it probably makes sense to just use Prod.